sql/dd/impl/bootstrap/bootstrapper.cc

```diff
@@ -1072,6 +1072,27 @@ bool create_dd_schema(THD *thd) {
 }
 
 bool initialize_dd_properties(THD *thd) {
+  if (!opt_initialize) {
+    // Find out which ddse contains dd_properties table during restart
+    // Try rocksdb first
+    handlerton *rocksdb_ddse = ha_resolve_by_legacy_type(thd, DB_TYPE_ROCKSDB);
+    int error = HA_ERR_NO_SUCH_TABLE;
+    if (rocksdb_ddse != nullptr || rocksdb_ddse->table_exists_in_engine) {
```

Comment:
Important suggestion:
```suggestion
    if (rocksdb_ddse != nullptr && rocksdb_ddse->table_exists_in_engine) {
```

Less important suggestion (both subexpressions should use the same way for null comparison):
```suggestion
    if (rocksdb_ddse != nullptr && rocksdb_ddse->table_exists_in_engine != nullptr) {
```